docs/tutorials/Mobile-App-Shielding/Android-Tutorial.md

```diff
@@ -0,0 +1,65 @@
+# Implementing App Shielding on Android
+
+<!-- AUTHOR joshis_tweets 2020-06-22T00:00:00Z -->
+<!-- SIDEBAR _Sidebar_Android.md sticky -->
+
+<div id="banner">
+    <div class="alert alert-warning">
+        <strong>Personalized Configuration Required.</strong><br/>
+        <span>In order to follow this tutorial, you need to purchase the App Shielding technology by Wultra and have a tooling as well as custom configuration prepared by Wultra engineers. Both tooling and configuration is specifically designed for your application. Contact your sales representative or technical consultant in order to obtain the required components.</span>
+    </div>
+</div>
+
+In this tutorial, we will show you how to implement App Shielding in your Android app. This tutorial has three parts:
+
+- [Mobile App Shielding Overview](./Readme.md)
+- [App Shielding for iOS](./iOS-Tutorial.md)
+- **App Shielding for Android**
+
+## Prerequisites
+
+- App Shielding tools with the configuration prepared by Wultra.
+- Your app in the format of `*.apk`.
+- Build machine with Java 8 installation.
+- Keystore with the signing certificate.
+
+## Running From the Command-Line
+
+To apply App Shielding to an Android app using command-line, you can simply run the shielder script while providing the properties file:
```

Comment:
This should be reamerked with an information that this will work only on macOS and Linux (we may try it later on Windows Ubuntu subsystem _for science_, but my guess is that it doesn't work).